Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM localhost:5000/tradingbase
+LABEL "version"="0.1"
+LABEL "Description"="Trading Server"
+EXPOSE "9456/tcp"
+WORKDIR /app
+#Build the Protocol Buffers
+COPY protocolbuffers .
+RUN protocolbuffers/build.sh
+COPY python /app
+
+#Copy Database configuration
+COPY xml /app
+
+#Copy relevant files
```

Comment:
this is bad practice. you never want source code in an image, this is for 2 reasons
1. if the image is compromised so is your code
2. updating the code requires a full image rebuild

Rather build the image and have a file in the container that authenticates and then pulls the code
* for now authenticating by way of a username/password is fine however going forward when we have a cluster in the cloud that will need to be changed for something called an IAM role